scripts/seastar-addr2line

```diff
@@ -151,8 +151,8 @@ if args.test:
         ('/my/path+0x12f34', {'type': BacktraceResolver.BacktraceParser.Type.ADDRESS, 'prefix': None, 'addresses': [{'path': '/my/path', 'addr': '0x12f34'}]}),
         (' /my/path+0x12f34', {'type': BacktraceResolver.BacktraceParser.Type.ADDRESS, 'prefix': None, 'addresses': [{'path': '/my/path', 'addr': '0x12f34'}]}),
 
-        ('Some prefix 0x12f34', None),
-        ('Some prefix /my/path+0x12f34', None),
+        ('Some prefix 0x12f34', {'type': BacktraceResolver.BacktraceParser.Type.ADDRESS, 'prefix': None, 'addresses': [{'path': None, 'addr': '0x12f34'}]}),
+        ('Some prefix /my/path+0x12f34', {'type': BacktraceResolver.BacktraceParser.Type.ADDRESS, 'prefix': None, 'addresses': [{'path': '/my/path', 'addr': '0x12f34'}]}),
```

Comment:
right.  and it is invoked only manually.
it works well enough since the logic changes very infrequently
and currently I'm the only one changing it and I use the self-test to verify the changes.